orquesta/statuses.py

```diff
@@ -36,6 +37,7 @@
     PAUSING,
     PAUSED,
     RESUMING,
+    RERUNNING,
```

Comment:
Remove if these statuses are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.